desk/src/pages/desk/ticket/editor/BottomSection.vue

```diff
@@ -140,6 +142,19 @@ const insertRes = createResource({
 	onSuccess() {
 		clean();
 	},
+	validate(params) {
+		if (isEmpty(params.doc.content)) {
```

Comment:
We can use `isEmpty` from `tiptap` here too

Reply:
I tried, but it didn't work, I was still able to add an empty comment.

Reply:
I used this though `params.doc.tiptap?.isEmpty`

Reply:
I just went on using `editor` object anyway.